content/en/docs/tasks/inject-data-application/distribute-credentials-secure.md

```diff
@@ -335,6 +335,131 @@ This functionality is available in Kubernetes v1.6 and later.
   password: 39528$vdg7Jb
   ```
 
+## Example: Provide prod/test credentials to Pods using Secrets {#provide-prod-test-creds}
+
+This example illustrates a Pod which consumes a secret containing production credentials and
+another Pod which consumes a secret with test environment credentials.
+
+1. Create a secret for prod environment credentials:
+
+   ```shell
+   kubectl create secret generic prod-db-secret --from-literal=username=produser --from-literal=password=Y4nys7f11
+   ```
+
+   The output is similar to:
+
+   ```
+   secret "prod-db-secret" created
+   ```
+
+1. Create a secret for test environment credentials.
+
+   ```shell
+   kubectl create secret generic test-db-secret --from-literal=username=testuser --from-literal=password=iluvtests
+   ```
+
+   The output is similar to:
+
+   ```
+   secret "test-db-secret" created
+   ```
+
+    {{< note >}}
+    Special characters such as `$`, `\`, `*`, `=`, and `!` will be interpreted by your
+    [shell](https://en.wikipedia.org/wiki/Shell_(computing)) and require escaping.
+
+    In most shells, the easiest way to escape the password is to surround it with single quotes (`'`).
+    For example, if your actual password is `S!B\*d$zDsb=`, you should execute the command as follows:
```

Comment:
```suggestion
   {{< note >}}
   Special characters such as `$`, `\`, `*`, `=`, and `!` will be interpreted by your
   [shell](https://en.wikipedia.org/wiki/Shell_(computing)) and require escaping.

   In most shells, the easiest way to escape the password is to surround it with single quotes (`'`).
   For example, if your actual password is `S!B\*d$zDsb=`, you should execute the command as follows:
```
If possible, using a same indentations (3 spaces) as the previous lines is more appreciated :)